src/main.cpp

```diff
@@ -4323,12 +4330,15 @@ void GridcoinServices()
                 bDoTally_retired = true;
             }
 
+            if ((nBestHeight % 5)==0)
+            {
+                if (fDebug) printf("SVC: Updating Neural Supermajority (v3 D) height %d\n",nBestHeight);
+                UpdateNeuralNetworkQuorumData();
```

Comment:
I think u ment ComputeNeuralNetworkSupermajorityHashes();? here. this would cause the same outcome as i was working to fix if you used UpdateNeuralNetworkQuorumData();

Reply:
good catch, much appreciated